lib/linter.sh

```diff
@@ -1677,6 +1691,26 @@ if [ "${VALIDATE_PYTHON_FLAKE8}" == "true" ]; then
   LintCodebase "PYTHON_FLAKE8" "flake8" "flake8 --config=${PYTHON_FLAKE8_LINTER_RULES}" ".*\.\(py\)\$" "${FILE_ARRAY_PYTHON_FLAKE8[@]}"
 fi
 
+################
+# R LINTING #
+################
+if [ "${VALIDATE_R}" == "true" ]; then
+  ##########################
+  # Check for local config #
+  ##########################
+  if [ ! -f "${GITHUB_WORKSPACE}/.lintr" ]; then 
+    info " "
+    info "No .lintr configuration file found, using defaults."
+    cp $R_LINTER_RULES "$GITHUB_WORKSPACE"
+  fi
+
+  ######################
+  # Lint the R files   #
+  ######################
+  # LintCodebase "FILE_TYPE" "LINTER_NAME" "LINTER_CMD" "FILE_TYPES_REGEX" "FILE_ARRAY"
+  LintCodebase  "R" "lintr" "lintr::lint('$File')" ".*\.\(r\|R\|Rmd\|rmd\)\$" "${FILE_ARRAY_R[@]}"
```

Comment:
```suggestion
LintCodebase  "R" "lintr" "lintr::lint('')" ".*\.\(r\|rmd\)\$" "${FILE_ARRAY_R[@]}"
```

The file type is done in all lower so no need to look for options, and then can you run the cammonad like that?

Reply:
Are you sure? I tried it with only lower case and no files where linted.